airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionUpdaterWorkflow.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.scheduling;
+
+import io.temporal.workflow.SignalMethod;
+import io.temporal.workflow.WorkflowInterface;
+import io.temporal.workflow.WorkflowMethod;
+
+@WorkflowInterface
+public interface ConnectionUpdaterWorkflow {
```

Comment:
Some javadocs for this interface would be helpful to understand what the goal of this workflow is and what its inputs/outputs are

Reply:
Thanks, I will add them. I will need to validate that the workflow is what we want to do with Jared and then I will add some doc.